Assets/Project/Scripts/Modules/GamePlayScene/Tile/TileGenerator.cs

```diff
@@ -22,26 +23,14 @@ public class TileGenerator : SingletonObject<TileGenerator>
 
         public UniTask CreateTiles(ICollection<TileData> tileDatas)
         {
-            // シーンに配置したノーマルタイルを初期化
-            for (var tileNum = 1; tileNum <= Constants.StageSize.ROW * Constants.StageSize.COLUMN; ++tileNum) {
-                var currTileObj = transform.Find($"NormalTile{tileNum}");
-                if (currTileObj == null) continue;
-
-                var currTile = currTileObj.GetComponent<NormalTileController>();
-                if (currTile == null) continue;
-
-                // initialize tile
-                BoardManager.Instance.SetTile(currTile, tileNum);
-                currTile.Initialize(tileNum);
-
-                // show sprite
-                currTile.GetComponent<SpriteRenderer>().enabled = true;
-            }
+            // 生成する特殊Tileの番号リスト
+            var tileNumList = tileDatas.Select(tileData => tileData.number).Concat(tileDatas.Select(tileData => tileData.pairNumber));
 
             var tasks = tileDatas
                 .Where(tileData => tileData.type == ETileType.Warp)
                 .Select(tileData => CreateWarpTiles(tileData.number, tileData.pairNumber))
                 .Concat(
+                    // WarpTile以外の特殊Tileの生成
                     tileDatas
                         .Where(tileData => tileData.type == ETileType.Holy || tileData.type == ETileType.Spiderweb || tileData.type == ETileType.Ice)
```

Comment:
```suggestion
                        .Where(tileData.type != ETileType.Warp && tileData.type != ETile.Normal)
```
タイル種類が増えた時に考えるとこの方が変更忘れがなさそう

Reply:
bd8a291